coog/templates/customer-backend/deployment.yaml

```diff
@@ -38,8 +41,14 @@ spec:
             procMount: Default
             runAsNonRoot: true
             runAsUser: 1000
+          resources:
+            {{- toYaml .Values.coog.initContainers.resources | nindent 12 }}
```

Comment:
Do you really want to have same resources than Coog init container ?

Reply:
My bad, bad c/c